CONTRIBUTING.md

```diff
@@ -31,7 +31,7 @@ To send us a pull request, please:
 
 1. Fork the repository.
 2. Modify the source; please focus on the specific change you are contributing. If you also reformat all the code, it will be hard for us to focus on your change.
-3. Ensure local tests pass.
+3. Ensure local tests pass by running `yarn test -u`.
```

Comment:
We could be a little bit more descriptive here. Could you add a bit that mentions why we need to add the `-u` tag? I.e., what it does and why do the snapshots need updating

Reply:
Thanks for the input @sejli! Just added more details per your suggestions.